src/test/features/response/helpers/taskListBuilder.ts

```diff
@@ -99,15 +99,18 @@ describe('Defendant response task list builder', () => {
     })
 
     describe('"Check and submit your response" task', () => {
+      let stub: sinon.SinonStub
       let isResponseRejectedFullyWithDisputePaidStub: sinon.SinonStub
       let isResponseRejectedFullyWithAmountClaimedPaidStub: sinon.SinonStub
 
       beforeEach(() => {
+        stub = sinon.stub(ResponseDraft.prototype, 'isResponsePopulated')
```

Comment:
Please rename it to something more meaningful. To follow convention I would suggest `isResponsePopulatedStub `